.github/workflows/submit.yml

```diff
@@ -0,0 +1,274 @@
+#
+# Copyright (c) 2020, Oracle and/or its affiliates. All rights reserved.
+# DO NOT ALTER OR REMOVE COPYRIGHT NOTICES OR THIS FILE HEADER.
+#
+# This code is free software; you can redistribute it and/or modify it
+# under the terms of the GNU General Public License version 2 only, as
+# published by the Free Software Foundation.  Oracle designates this
+# particular file as subject to the "Classpath" exception as provided
+# by Oracle in the LICENSE file that accompanied this code.
+#
+# This code is distributed in the hope that it will be useful, but WITHOUT
+# ANY WARRANTY; without even the implied warranty of MERCHANTABILITY or
+# FITNESS FOR A PARTICULAR PURPOSE.  See the GNU General Public License
+# version 2 for more details (a copy is included in the LICENSE file that
+# accompanied this code).
+#
+# You should have received a copy of the GNU General Public License version
+# 2 along with this work; if not, write to the Free Software Foundation,
+# Inc., 51 Franklin St, Fifth Floor, Boston, MA 02110-1301 USA.
+#
+# Please contact Oracle, 500 Oracle Parkway, Redwood Shores, CA 94065 USA
+# or visit www.oracle.com if you need additional information or have any
+# questions.
+#
+
+name: JavaFX pre-submit tests
+
+on:
+  push:
+    branches-ignore:
+      - master
```

Comment:
Add `jfx[0-9]*` to this. Also consider adding `WIP.*`, `scratch.*`, and similar.

Reply:
I propose to ignore the following branches:

```
    branches-ignore:
      - master
      - main
      - 'jfx[0-9]+'
      - 'WIP*'
```

The thinking behind this is that you will still get build / test by default, but if you are are working on something experimental and don't want the noise of an automatic build / test on every push to your branch, you can prefix your branch name with "WIP".